### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lindong28**
## What is the purpose of the change

Update README to better clarify the benefits of FeatHub. 

## Brief change log

- Updated the Introduction and the Benefits sections in README.
- Updated architecture figures.
- Re-organized docs and added the table of content in `docs/content/_index.md`.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by lindong28** [comment]
@Sxnan Can you help review this PR?

File: README.md [hunk 1]
```diff
@@ -1,123 +1,129 @@
...
-architecture.
-
-<img src="docs/figures/architecture_2.png" width="70%" height="auto">
-
-The figure above shows how a developer can use FeatHub to facilitate feature
-engineering for model training and model inference. Please checkout [Feathub
-architecture and concepts](docs/architecture.md) for more details.
-
-## Getting Started
+FeatHub is a stream-batch unified feature store that simplifies feature
```
Comment (Sxnan) [inline README.md:1]: Should we rename all the `Feathub` to `FeatHub`? Or we can create an issue to do that late.

File: README.md [hunk 2]
```diff
@@ -1,123 +1,129 @@
...
+development, deployment, monitoring, and sharing for machine learning
+applications.
+
+- [What is FeatHub](#what-is-feathub)
```
Comment (Sxnan) [inline README.md:5]: I think this section is not in the docs. And we are missing the `Introduction` and `Core Benefits` sections.

**Review by Sxnan — COMMENTED** [review]
Thanks for your PR! Just left two minor comments.

File: README.md (hunk 1 above)
Comment (lindong28) [reply README.md:1]: Thanks for the suggestion. It is fixed now.

File: README.md (hunk 2 above)
Comment (lindong28) [reply README.md:5]: Thanks for catching this. It is fixed now.

**Comment by lindong28** [comment]
@Sxnan Thanks for the review! I have updated the PR as suggested.

**Comment by Sxnan** [comment]
@lindong28 Thanks for your update! LGTM.

**Pull request closed without merging**